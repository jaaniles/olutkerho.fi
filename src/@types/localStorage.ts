export enum LocalStorage {
  LANG = "LANG",
  DARKMODE = "DARKMODE",
}
