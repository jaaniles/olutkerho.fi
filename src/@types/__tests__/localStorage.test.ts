import { LocalStorage } from "../localStorage";

it("Check LocalStorage", () => {
  expect.assertions(1);

  return expect(LocalStorage).toMatchSnapshot();
});
