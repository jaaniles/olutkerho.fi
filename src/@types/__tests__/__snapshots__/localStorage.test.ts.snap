// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Check LocalStorage 1`] = `
Object {
  "DARKMODE": "DARKMODE",
  "LANG": "LANG",
}
`;
